R/dummy.R

```diff
@@ -314,6 +314,9 @@ bake.step_dummy <- function(object, new_data, ...) {
         data = indicators
       )
     indicators <- as_tibble(indicators)
+    if (fac_type != "ordered") {
+      indicators <- purrr::map_dfc(indicators, vec_cast, integer())
```

Comment:
Good idea, lets put that work into a separate issue since `list_cbind` isn't available yet. This way we can catch everything in one fell swoop.

I would imagine that `map() |> list_cbind()` is going to preferred over `map |> bind_cols()`